reference/sectionD_references.inc

```diff
@@ -0,0 +1,17 @@
+
+.. _sectionD:
+
+Extensible Markup Language (XML) 1.1 (Second Edition) http://www.w3.org/TR/2006/REC-xml11-20060816 (16 August 2006)
```

Comment:
Prefer https://www.w3.org/TR/2006/REC-xml11-20060816/. Otherwise 29 September 2006.

Reply:
See url - date looks like August 16?

Reply:
Sure, but edited in place on 29 September 2006. @nickerso and @hsorby, views?

Reply:
as for MathML (and all these URLs), should be using the canonical reference not a versioned instance: https://www.w3.org/TR/xml11/. And if you want to include a date it would be the date accessed. But I'd suggest not putting a date.

Reply:
Ok for not putting a date. After all, the URL is really all we need (and agreed re using the canonical reference).